src/messages/components/ConversationAddReactionInner.vue

```diff
@@ -17,24 +20,73 @@
 
 <script>
 import EmojiButton from './EmojiButton'
+import emojiList from 'markdown-it-emoji/lib/data/full.json'
+import { QSearch } from 'quasar'
+
+function searchEmoji (search) {
+  // Remove colons and set to lowercase to normalize search
+  let cleanedSearch = search.replace(/:/g, '').toLowerCase()
+  let matchingEmojiNames = []
+  // The set is used to remove duplicate emoji that may have different names
+  let matchingEmojiUnicodes = new Set()
+  // First add the exact search in case there is an emoji shortcode with less
+  // than three characters
+  if (Object.keys(emojiList).includes(cleanedSearch)) {
+    matchingEmojiUnicodes.add(emojiList[cleanedSearch])
+    matchingEmojiNames.push(cleanedSearch)
+  }
+
+  // Checks the length because we don't want to have too many results
+  if (cleanedSearch.length > 2) {
+    for (var index in Object.keys(emojiList)) {
```

Comment:
Since you don't need the index, what about using `emojiList.forEach()`?

Reply:
`forEach()` didn't let us return when the emoji matches reached 20, but I scrapped using the index anyways by looping over `emojiList` instead of over `Object.keys(emojiList)`